tools/oeutil/README.md

```diff
@@ -1,19 +1,29 @@
-oegenerate
+OEUTIL
 =====
 
-`oegenerate` is a debugging utility that generates the following files in binary format, or dump them in readable text, and verify them:
+`oeutil` integrates following OE utilities:
+
+    1. generate-evidence
+
+Usage: `oeutil <util> <util options>`
```

Comment:
Add underscore between util and options (i.e. util_option) to indicate it's one token

Reply:
Make sense. Updated.